bindata/network/ovn-kubernetes/self-hosted/common/004-config.yaml

```diff
@@ -61,6 +61,12 @@ data:
 {{- if (index . "V6JoinSubnet") }}
     v6-join-subnet="{{.V6JoinSubnet}}"
 {{- end }}
+
+    [logging]
+    libovsdblogfile=/var/log/ovnkube/libovsdb.log
+    logfile-maxsize=100
+    logfile-maxbackups=5
+    logfile-maxage=0
```

Comment:
@flavio-fernandes  It looks like this configuration will also affect the CNI logging. Just making sure we are aware and that's ok

https://github.com/ovn-org/ovn-kubernetes/blob/8055a47068e0689d339e6b8a66922fe809e2afe9/go-controller/pkg/config/cni.go#L32-L34

Reply:
yup, it is ok, iirc.